onnx/reference/ops/aionnx_preview_training/_op_run_training.py

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 # pylint: disable=R0913,W0221
 
-from ..op_run import OpRun
+from ...op_run import OpRun
```

Comment:
Possible to use absolute module paths?

Reply:
[PEP8](https://peps.python.org/pep-0008/#imports) recommends: 

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path):

```
import mypkg.sibling
from mypkg import sibling
from mypkg.sibling import example
```

imo it improves readability (tells reader where the module comes from without having to go back to the dir structure) and makes the code easier to maintain when we move files around. Things start to get complicated when we have lines like `...foo.bar` or `....foo` and we may quickly lose count of the dots.

Reply:
I did not know. I usually use relative imports because when I work on a package and the package is also installed in site-package, absolute imports are ambiguous. I was searching for speeds but did not find anything about importing time. I assumed that relative import are faster since there is only one possible location instead of having mulitple (sys.path). But maybe I'm wrong. onnx.reference.ops._op_list imports more than 100 files, maybe that's worth measuring before changing everything.

Reply:
Without having done the measurement myself, imported modules are cached so I think any speed difference would only matter the first time (and the difference should be small anyways)?

For site-package ambiguity, does developing in venv help?

Reply:
No worries, it is just a question of habits. I've been using relative imports since I discovered them. onnx is using absolute import in all other places, so I should use absolute too.

Reply:
done